sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/implementation/SpringAppDeploymentImpl.java

```diff
@@ -144,12 +140,25 @@ private void ensureDeploySettings() {
         }
     }
 
-    private void ensureSource() {
+    private void ensureSource(UserSourceType type) {
         if (innerModel().properties() == null) {
             innerModel().withProperties(new DeploymentResourceProperties());
         }
         if (innerModel().properties().source() == null) {
-            innerModel().properties().withSource(new UserSourceInfo());
+            switch (type.toString()) {
```

Comment:
Prefer a "if else".

toString have unnecessary risk of server change the value e.g. Jar to JAR

Reply:
They actually deleted this class somehow...